index.html

```diff
@@ -147,6 +147,16 @@
 problems you set out to solve when designing your feature(s). How does the
 threat model change? What unexpected consequences could arise?
 
+<aside class="example" title="Misuse" id="example-misuse">
+
+    Example: A bad actor could implement the [Web Speech
+    API](https://webaudio.github.io/web-speech-api/) in order to eavesdrop
+    on web users. To prevent this, the authors of the spec have included the
+    requirement for user agents to give the web user an obvious
+    indication when audio is being recorded.
```

Comment:
```suggestion
    Example: A bad actor could implement the [Compute Pressure API
    API](https://www.w3.org/TR/compute-pressure/) to compromise the anonymity
    of web users. To prevent this, the authors of the spec have included the
    requirement for user agents to minimize data exposure and implement rate
    obfuscation techniques to prevent side-channel information leaks.
```

Reply:
s/implement/use/ since the bad actor we're worried about is a web page, not a browser.

Also, less importantly, I think s/compromise the anonymity of web users/join web user identities across sites/, since that covers de-anonymization but also includes users who only intend to be pseudonymous.

But I think this is overall an important change to make.